javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,77 @@
+// Age-ify (A future age calculator)
+const yearOfBirth = 1986;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
+
+//Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let humanYear = dogYearFuture - dogYearOfBirth;
+let dogYear = humanYear * 7;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true) {
+  console.log(
+    "Your dog will be " +
+      humanYear +
+      " human years old in " +
+      dogYearFuture +
+      "."
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture + "."
+  );
+}
```

Comment:
Great structure and once again good attention to detail. 👌

OBS! 😊
Have a second look at your boolean variable shouldShowResultInDogYears. What are you printing out if this value is true?

Reply:
I have mixed up the true and false statements. I will make it to be true and show the result in dog years.